src/index.ts

```diff
@@ -123,6 +123,18 @@ class Monika extends Command {
       required: false,
     }),
 
+    symonReportInterval: Flags.string({
+      description: 'Interval for reporting to Symon (optional)',
```

Comment:
please add the unit of the interval in the description, is it `seconds` or `milliseconds`